src/main/resources/xslt/preservica2solr.xsl

```diff
@@ -76,7 +150,6 @@
           </f:string>
         </xsl:for-each>
       </f:array>
-
       <xsl:for-each select="pbcoreGenre/genre">
         <xsl:choose>
           <xsl:when test="f:contains(., 'hovedgenre:')">
```

Comment:
`for-each` indicates that there can be more than one genre, but in the `schema.xml`, `genre` is single-valued? If Preservica guarantees that there is only a single `hovedgenre` and a single `undergenre`, please state this in a comment in the XSLT.

Reply:
I'll add the comment. Just out of curiosity, would you prefer two if statements instead?

Reply:
No. `choose` is equivalent to a `switch` and that signals a "trivial" check between a finite list of match-values. `if` is open and signals a more flexible approach where secondary considerations are possible. But we're really at the finer points of conditional choosing here.